stdlib/public/stubs/Stubs.cpp

```diff
@@ -252,8 +258,43 @@ extern "C" uint64_t swift_float80ToString(char *Buffer, size_t BufferLength,
 /// if an error occurred, or EOF was reached.
 SWIFT_RUNTIME_STDLIB_INTERFACE
 extern "C" ssize_t swift_stdlib_readLine_stdin(char **LinePtr) {
+#if defined(_MSC_VER)
+  if (LinePtr == nullptr)
+    return -1;
+
+  int Capacity = 0;
+  ssize_t Pos = 0;
+  char *ReadBuf = *LinePtr;
+
+  for (;;) {
+    int ch = getc(stdin);
+    if (ferror(stdin))
+      return -1;
+    if (ch == EOF && Pos == 0)
+      return -1;
+
+    if (Capacity - Pos <= 1) {
+      // Capacity changes to 128, 128*2, 128*4, 128*8, ...
+      Capacity = Capacity ? Capacity * 2 : 128;
+      ReadBuf = static_cast<char *>(realloc(ReadBuf, Capacity));
```

Comment:
this leaks memory if realloc fails after first iteration.   not sure why you changed this func away from using std::getline?